test-cases/longevity/longevity-10gb-3h.yaml

```diff
@@ -13,3 +13,9 @@ nemesis_interval: 5
 
 user_prefix: 'longevity-10gb-3h'
 space_node_threshold: 64424
+ip_ssh_connections: "ipv6"
```

Comment:
Not sure we want it in this longevity specifically, and if so we may want to add it in the name.